website/content/docs/license/utilization/manual-reporting.mdx

```diff
@@ -41,6 +41,16 @@ reporting in favor of manual reporting.
 Data bundles include collections of JSON snapshots that contain license
 utilization information.
 
+<Tip title="Vault Policy Requirement">
+To run this command, you will need a Vault policy with at least the following:
```

Comment:
```suggestion
<Note title="Vault policy requirement">

To run the reporting command, you must have a Vault policy that grants
`update` permission on the `sys/utilization` endpoint:
```

Markdown correction: aside tags (e.g., `<Tip>`) require whitespace before and after so the parser doesn't get confused

Style correction: avoid "this" as a pronoun, write in declarative statements.

Reply:
Tips should always be optional "this can make your life easier" information. Since the endpoint has specific policy permissions required to call it successfully, we should make the aside a note instead of a tip

Reply:
Aha! Yes, a useful distinction.

And thanks for the style guide notes!